docs/internals/howto-release-django.txt

```diff
@@ -364,48 +429,63 @@ OK, this is the fun part, where we actually push out a release!
    ``Django-<version>.checksum.txt.asc`` which you can then verify using ``gpg
    --verify Django-<version>.checksum.txt.asc``.
 
-If you're issuing multiple releases, repeat these steps for each release.
-
 Making the release(s) available to the public
 =============================================
 
 Now you're ready to actually put the release out there. To do this:
 
-#. Upload the release package(s) to the djangoproject server, replacing
-   A.B. with the appropriate version number, e.g. 4.1 for a 4.1.x release:
+#. Upload the checksum file(s):
```

Comment:
I moved this to an earlier step because it can be done at any earlier point, even in the context of a security release (whilst uploading the binaries need to happen as close to the release time as possible).